src/Serilog.Sinks.ApplicationInsights/Sinks/ApplicationInsights/TelemetryConverters/TelemetryConverterBase.cs

```diff
@@ -39,6 +39,26 @@ public abstract class TelemetryConverterBase : ITelemetryConverter
     /// </summary>
     public const string OperationIdProperty = "operationId";
 
+    /// <summary>
+    ///     Property that is included when in log context, will be pushed out as AI trace id.
+    /// </summary>
+    public const string TraceIdProperty = "TraceId";
+
+    /// <summary>
+    ///     Property that is included when in log context, will be pushed out as AI parent span id.
+    /// </summary>
+    public const string ParentSpanIdProperty = "ParentSpanId";
```

Comment:
`ParentSpanId` does appear as a log event property, but `TraceId` and `SpanId` don't - they're always directly (and correctly) attached to the `LogEvent` so shouldn't need constants here.

Reply:
ACK. Removed it.